### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by TheAndrewCook**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@TheAndrewCook__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@TheAndrewCook__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@TheAndrewCook Can you sign the CLA to enable us to review and merge? If you're MSFT FTE, please send me a mail offline

File: 202-site-to-site-vpn/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
```
Comment (singhkay) [inline 202-site-to-site-vpn/azuredeploy.json]: lower case "string". I think CLI fails due to this

File: 202-site-to-site-vpn/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+            "metadata": {
+                "description": "Region where the resources will be deployed"
+            },
+            "allowedValues": [
+                "East US",
```
Comment (singhkay) [inline 202-site-to-site-vpn/azuredeploy.json]: I think VPN might be available in other regions as well

File: 202-site-to-site-vpn/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                "West US",
+                "South Central US",
+                "Central US"
+            ]
+        },
+        "vpnType": {
+            "type": "String",
```
Comment (singhkay) [inline 202-site-to-site-vpn/azuredeploy.json]: lower case "string". I think CLI fails due to this

**Comment by singhkay** [comment]
Closing since the other PR #787 is merged

**Pull request closed without merging**